example/SliderExample.js

```diff
@@ -171,6 +174,20 @@ export const examples = [
       return <SliderExample thumbImage={require('./uie_thumb_big.png')} />;
     },
   },
+  {
+    title: 'Custom thumb (network image)',
+    platform: 'windows',
+    render(): Element<any> {
+      return (
+        <SliderExample
+          thumbImage={{
+            uri:
+              'https://cdn2.iconfinder.com/data/icons/minimalism/512/twitter.png',
```

Comment:
maybe use an icon in the open domain as opposed to an icon with a license attached to it (plus this link is in a CDN so it may change?)

Reply:
will do